ldi-orchestrator/ldio-connectors/ldio-console-out/pom.xml

```diff
@@ -8,6 +8,7 @@
     <modelVersion>4.0.0</modelVersion>
 
     <artifactId>ldio-console-out</artifactId>
+    <packaging>pom</packaging>
```

Comment:
please remove this. This will make this module only output a pom file, no jars aka no ldio modules to use